webapp/app/forms/steps/lotse/pauschbetrag.py

```diff
@@ -1,16 +1,41 @@
-def calculate_pauschbetrag(has_pflegegrad=False, disability_degree=None, has_merkzeichen_bl=False, has_merkzeichen_tbl=False, has_merkzeichen_h=False):
+from typing import Optional
+
+from flask import render_template
+from flask_babel import lazy_gettext as _l, ngettext, _
+from pydantic import BaseModel, root_validator, ValidationError, validator
+from wtforms.validators import InputRequired
+from wtforms import SelectField
+from flask_wtf.csrf import generate_csrf
+
+from app.forms.steps.lotse.fahrkostenpauschbetrag import calculate_fahrkostenpauschbetrag
+from app.model.components import PauschbetragProps
+from app.model.components.helpers import form_fields_dict
+from app.forms import SteuerlotseBaseForm
+from app.forms.steps.step import SectionLink
+from app.forms.steps.lotse.lotse_step import LotseFormSteuerlotseStep
+from app.forms.steps.lotse.personal_data import PersonAHasDisabilityPrecondition, PersonBHasDisabilityPrecondition, ShowPersonBPrecondition, StepFamilienstand, get_number_of_users
+
+
+from app.forms.steps.lotse.personal_data import get_number_of_users
+from app.model.components import NoPauschbetragProps
+
+
+def calculate_pauschbetrag(has_pflegegrad: str = None, disability_degree: int = None, has_merkzeichen_bl: bool = False,
```

Comment:
If we set the disability degree to zero here, we do not need to do the check below